README.md

```diff
@@ -135,6 +135,7 @@ To run as a desktop app:
 
    ```
    npm install electron
+   npm run install:electron
    node_modules/.bin/electron .
```

Comment:
@dbkr Matthew had a complaint when I did had linked installing these deps to postinstall: https://github.com/vector-im/riot-web/pull/3622#discussion_r112800196